autofill/autofill-api/src/main/java/com/duckduckgo/autofill/api/AutofillCredentialDialogs.kt

```diff
@@ -102,6 +102,32 @@ interface CredentialUpdateExistingCredentialsDialog {
     }
 }
 
+interface EmailProtectionChooserDialog {
```

Comment:
My bad! Because I was converting the existing dialog to use the autofill-style, it didn't register to me that it was a new public API.

Have documented now.